service_taxi/src/main/kotlin/org/thechance/service_taxi/domain/usecase/ManageTaxiUseCase.kt

```diff
@@ -48,15 +49,19 @@ class ManageTaxiUseCase(
     }
 
     override suspend fun editTaxi(taxiId: String,taxi: Taxi): Taxi {
-        validationTaxi(taxi)
+        validateTaxi(taxi)
         return taxiGateway.editTaxi(taxiId,taxi)
     }
 
     override suspend fun getNumberOfTaxis(): Long {
         return taxiGateway.getNumberOfTaxis()
     }
 
-    private fun validationTaxi(taxi: Taxi) {
+    override suspend fun isTaxiExistedBefore(taxi: Taxi): Boolean {
```

Comment:
It's not a use case. It's step in a use case flow. So we can make it 'private fun'